docs/Installation.md

```diff
@@ -1,117 +1,115 @@
-install Python with additional dependencies. Each of the subsections below
-overviews each step, in addition to a Docker set-up.
-
-## Install **Unity 2018.4** or Later
-
-[Download](https://store.unity.com/download) and install Unity. If you would
-like to use our Docker set-up (introduced later), make sure to select the _Linux
-Build Support_ component when installing Unity.
-
-<p align="center">
-  <img src="images/unity_linux_build_support.png"
-       alt="Linux Build Support"
-       width="500" border="10" />
-</p>
-
-## Environment Setup
-We now support a single mechanism for installing ML-Agents on Mac/Windows/Linux using Virtual
-Environments. For more information on Virtual Environments and installation instructions,
-follow this [guide](Using-Virtual-Environment.md).
+The ML-Agents Toolkit contains several components:
+* Unity package ([`com.unity.ml-agents`](../com.unity.ml-agents/)) contains the Unity C#
+SDK that will be integrated into your Unity scene.
+* Three Python packages:
+  * [`mlagents`](../ml-agents/) contains the machine learning algorithms that enables you
+  to train behaviors in your Unity scene. Most users of ML-Agents will only need to
+  directly install `mlagents`.
+  * [`mlagents_envs`](../ml-agents-envs/) contains a Python API to interact with a Unity
+  scene. It is a foundational layer that facilitates data messaging between Unity scene
+  and the Python machine learning algorithms. Consequently, `mlagents` depends on `mlagents_envs`.
+  * [`gym_unity`](../gym-unity/) provides a Python-wrapper for your Unity scene that
+  supports the OpenAI Gym interface.
+* Unity [Project](../Project/) that contains several
+[example environments](Learning-Environment-Examples.md) that highlight the various features
+of the toolkit to help you get started.
+
+Consequently, to install and use ML-Agents you will need to:
+* Install Unity (2018.4 or later)
+* Install Python (3.6.1 or higher)
+* Clone this repository
+* Install the `com.unity.ml-agents` Unity package
+* Install the `mlagents` Python package
+
+### Install **Unity 2018.4** or Later
+
+[Download](https://unity3d.com/get-unity/download) and install Unity. We strongly recommend
+that you install Unity through the Unity Hub as it will enable you to manage multiple Unity
+versions.
+
+### Install **Python 3.6.1** or Higher
+
+We recommend [installing](https://www.python.org/downloads/) Python 3.6 or 3.7. If your Python
+environment doesn't include `pip3`, see these
+[instructions](https://packaging.python.org/guides/installing-using-linux-tools/#installing-pip-setuptools-wheel-with-linux-package-managers)
+on installing it.
 
-Although we don't support Anaconda installation path of ML-Agents for Windows, the previous guide
-is still in the docs folder.  Please refer to [Windows Installation (Deprecated)](Installation-Windows.md).
+Although we do not support the Anaconda installation path for Windows, the previous
```

Comment:
```suggestion
Although we do not provide support for Anaconda installation on Windows, the previous
```
"do not support" implies it does work, trying to go for "it might work but we won't help you with it."